Update few `fast/ruby` tests to 'reftests'

Ahmad-S792 opened the pull request:
#### 21879fdc9bb7c122e404d0ba754e881b68ab5d9e
<pre>
Update few `fast/ruby` tests to &apos;reftests&apos;

<a href="https://bugs.webkit.org/show_bug.cgi?id=275854">https://bugs.webkit.org/show_bug.cgi?id=275854</a>

Reviewed by Alan Baradlay.

Merge: <a href="https://chromium-review.googlesource.com/c/chromium/src/+/2272260">https://chromium-review.googlesource.com/c/chromium/src/+/2272260</a>

* LayoutTests/platform/glib/fast/ruby/rubyDOM-insert-rt-expected.png: Removed.
* LayoutTests/platform/glib/fast/ruby/rubyDOM-insert-rt-expected.txt: Removed.
* LayoutTests/platform/glib/fast/ruby/rubyDOM-insert-text1-expected.png: Removed.
* LayoutTests/platform/glib/fast/ruby/rubyDOM-insert-text1-expected.txt: Removed.
* LayoutTests/platform/glib/fast/ruby/rubyDOM-insert-text2-expected.png: Removed.
* LayoutTests/platform/glib/fast/ruby/rubyDOM-insert-text2-expected.txt: Removed.
* LayoutTests/platform/glib/fast/ruby/rubyDOM-insert-text3-expected.png: Removed.
* LayoutTests/platform/glib/fast/ruby/rubyDOM-insert-text3-expected.txt: Removed.
* LayoutTests/platform/glib/fast/ruby/rubyDOM-remove-rt1-expected.png: Removed.
* LayoutTests/platform/glib/fast/ruby/rubyDOM-remove-rt1-expected.txt: Removed.
* LayoutTests/platform/glib/fast/ruby/rubyDOM-remove-rt2-expected.png: Removed.
* LayoutTests/platform/glib/fast/ruby/rubyDOM-remove-rt2-expected.txt: Removed.
* LayoutTests/platform/glib/fast/ruby/rubyDOM-remove-text1-expected.png: Removed.
* LayoutTests/platform/glib/fast/ruby/rubyDOM-remove-text1-expected.txt: Removed.
* LayoutTests/platform/glib/fast/ruby/rubyDOM-remove-text2-expected.png: Removed.
* LayoutTests/platform/glib/fast/ruby/rubyDOM-remove-text2-expected.txt: Removed.
* LayoutTests/platform/ios/fast/ruby/rubyDOM-insert-rt-expected.txt: Removed.
* LayoutTests/platform/ios/fast/ruby/rubyDOM-insert-text1-expected.txt: Removed.
* LayoutTests/platform/ios/fast/ruby/rubyDOM-insert-text2-expected.txt: Removed.
* LayoutTests/platform/ios/fast/ruby/rubyDOM-insert-text3-expected.txt: Removed.
* LayoutTests/platform/ios/fast/ruby/rubyDOM-remove-rt1-expected.txt: Removed.
* LayoutTests/platform/ios/fast/ruby/rubyDOM-remove-rt2-expected.txt: Removed.
* LayoutTests/platform/ios/fast/ruby/rubyDOM-remove-text1-expected.txt: Removed.
* LayoutTests/platform/ios/fast/ruby/rubyDOM-remove-text2-expected.txt: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-insert-rt-expected.png: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-insert-rt-expected.txt: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-insert-text1-expected.png: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-insert-text1-expected.txt: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-insert-text2-expected.png: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-insert-text2-expected.txt: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-insert-text3-expected.png: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-insert-text3-expected.txt: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-remove-rt1-expected.png: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-remove-rt1-expected.txt: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-remove-rt2-expected.png: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-remove-rt2-expected.txt: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-remove-text1-expected.png: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-remove-text1-expected.txt: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-remove-text2-expected.png: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-remove-text2-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/ruby/rubyDOM-insert-rt-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/ruby/rubyDOM-insert-text1-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/ruby/rubyDOM-insert-text2-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/ruby/rubyDOM-insert-text3-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/ruby/rubyDOM-remove-rt1-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/ruby/rubyDOM-remove-rt2-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/ruby/rubyDOM-remove-text1-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/ruby/rubyDOM-remove-text2-expected.txt: Removed.
* LayoutTests/fast/ruby/rubyDOM-insert-rt-expected.html: Add Test Case Expectation
* LayoutTests/fast/ruby/rubyDOM-insert-text1-expected.html: Ditto
* LayoutTests/fast/ruby/rubyDOM-insert-text2-expected.html: Ditto
* LayoutTests/fast/ruby/rubyDOM-insert-text3-expected.html: Ditto
* LayoutTests/fast/ruby/rubyDOM-remove-rt1-expected.html: Ditto
* LayoutTests/fast/ruby/rubyDOM-remove-rt2-expected.html: Ditto
* LayoutTests/fast/ruby/rubyDOM-remove-text1-expected.html: Ditto
* LayoutTests/fast/ruby/rubyDOM-remove-text2-expected.html: Ditto
* LayoutTests/fast/ruby/rubyDOM-insert-rt.html: Update Test Case
* LayoutTests/fast/ruby/rubyDOM-insert-text1.html: Ditto
* LayoutTests/fast/ruby/rubyDOM-insert-text2.html: Ditto
* LayoutTests/fast/ruby/rubyDOM-insert-text3.html: Ditto
* LayoutTests/fast/ruby/rubyDOM-remove-rt1.html: Ditto
* LayoutTests/fast/ruby/rubyDOM-remove-rt2.html: Ditto
* LayoutTests/fast/ruby/rubyDOM-remove-text1.html: Ditto
* LayoutTests/fast/ruby/rubyDOM-remove-text2.html: Ditto

Canonical link: <a href="https://commits.webkit.org/280342@main">https://commits.webkit.org/280342@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/8540f1192201a3b52997111b1ced3b1538c152a9

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/56345 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/35671 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/8817 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/59952 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/59/builds/6781 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/43293 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/6975 "Built successfully") | [⏳ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/WPE-WK2-Tests-EWS "Waiting to run tests") | [✅ 🧪 wincairo-tests](https://ews-build.webkit.org/#/builders/60/builds/4732 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/58374 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/33547 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/48613 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/26499 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/30326 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/5943 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/5785 "Built successfully") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/52324 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/6215 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/61636 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/87/builds/253 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/62/builds/6344 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/52901 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/86/builds/253 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/48679 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/52784 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/12475 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/88/builds/226 "Passed tests") | | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/31498 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/32584 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/33667 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/32331 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/f4f3d999be6fcfa81286f0724c62708de764f153. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/30150

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/8540f1192201a3b52997111b1ced3b1538c152a9. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/30150

Commit:
Update few `fast/ruby` tests to 'reftests'

https://bugs.webkit.org/show_bug.cgi?id=275854

Reviewed by Alan Baradlay.

Merge: https://chromium-review.googlesource.com/c/chromium/src/+/2272260

* LayoutTests/platform/glib/fast/ruby/rubyDOM-insert-rt-expected.png: Removed.
* LayoutTests/platform/glib/fast/ruby/rubyDOM-insert-rt-expected.txt: Removed.
* LayoutTests/platform/glib/fast/ruby/rubyDOM-insert-text1-expected.png: Removed.
* LayoutTests/platform/glib/fast/ruby/rubyDOM-insert-text1-expected.txt: Removed.
* LayoutTests/platform/glib/fast/ruby/rubyDOM-insert-text2-expected.png: Removed.
* LayoutTests/platform/glib/fast/ruby/rubyDOM-insert-text2-expected.txt: Removed.
* LayoutTests/platform/glib/fast/ruby/rubyDOM-insert-text3-expected.png: Removed.
* LayoutTests/platform/glib/fast/ruby/rubyDOM-insert-text3-expected.txt: Removed.
* LayoutTests/platform/glib/fast/ruby/rubyDOM-remove-rt1-expected.png: Removed.
* LayoutTests/platform/glib/fast/ruby/rubyDOM-remove-rt1-expected.txt: Removed.
* LayoutTests/platform/glib/fast/ruby/rubyDOM-remove-rt2-expected.png: Removed.
* LayoutTests/platform/glib/fast/ruby/rubyDOM-remove-rt2-expected.txt: Removed.
* LayoutTests/platform/glib/fast/ruby/rubyDOM-remove-text1-expected.png: Removed.
* LayoutTests/platform/glib/fast/ruby/rubyDOM-remove-text1-expected.txt: Removed.
* LayoutTests/platform/glib/fast/ruby/rubyDOM-remove-text2-expected.png: Removed.
* LayoutTests/platform/glib/fast/ruby/rubyDOM-remove-text2-expected.txt: Removed.
* LayoutTests/platform/ios/fast/ruby/rubyDOM-insert-rt-expected.txt: Removed.
* LayoutTests/platform/ios/fast/ruby/rubyDOM-insert-text1-expected.txt: Removed.
* LayoutTests/platform/ios/fast/ruby/rubyDOM-insert-text2-expected.txt: Removed.
* LayoutTests/platform/ios/fast/ruby/rubyDOM-insert-text3-expected.txt: Removed.
* LayoutTests/platform/ios/fast/ruby/rubyDOM-remove-rt1-expected.txt: Removed.
* LayoutTests/platform/ios/fast/ruby/rubyDOM-remove-rt2-expected.txt: Removed.
* LayoutTests/platform/ios/fast/ruby/rubyDOM-remove-text1-expected.txt: Removed.
* LayoutTests/platform/ios/fast/ruby/rubyDOM-remove-text2-expected.txt: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-insert-rt-expected.png: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-insert-rt-expected.txt: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-insert-text1-expected.png: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-insert-text1-expected.txt: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-insert-text2-expected.png: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-insert-text2-expected.txt: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-insert-text3-expected.png: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-insert-text3-expected.txt: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-remove-rt1-expected.png: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-remove-rt1-expected.txt: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-remove-rt2-expected.png: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-remove-rt2-expected.txt: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-remove-text1-expected.png: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-remove-text1-expected.txt: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-remove-text2-expected.png: Removed.
* LayoutTests/platform/mac/fast/ruby/rubyDOM-remove-text2-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/ruby/rubyDOM-insert-rt-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/ruby/rubyDOM-insert-text1-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/ruby/rubyDOM-insert-text2-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/ruby/rubyDOM-insert-text3-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/ruby/rubyDOM-remove-rt1-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/ruby/rubyDOM-remove-rt2-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/ruby/rubyDOM-remove-text1-expected.txt: Removed.
* LayoutTests/platform/wincairo/fast/ruby/rubyDOM-remove-text2-expected.txt: Removed.
* LayoutTests/fast/ruby/rubyDOM-insert-rt-expected.html: Add Test Case Expectation
* LayoutTests/fast/ruby/rubyDOM-insert-text1-expected.html: Ditto
* LayoutTests/fast/ruby/rubyDOM-insert-text2-expected.html: Ditto
* LayoutTests/fast/ruby/rubyDOM-insert-text3-expected.html: Ditto
* LayoutTests/fast/ruby/rubyDOM-remove-rt1-expected.html: Ditto
* LayoutTests/fast/ruby/rubyDOM-remove-rt2-expected.html: Ditto
* LayoutTests/fast/ruby/rubyDOM-remove-text1-expected.html: Ditto
* LayoutTests/fast/ruby/rubyDOM-remove-text2-expected.html: Ditto
* LayoutTests/fast/ruby/rubyDOM-insert-rt.html: Update Test Case
* LayoutTests/fast/ruby/rubyDOM-insert-text1.html: Ditto
* LayoutTests/fast/ruby/rubyDOM-insert-text2.html: Ditto
* LayoutTests/fast/ruby/rubyDOM-insert-text3.html: Ditto
* LayoutTests/fast/ruby/rubyDOM-remove-rt1.html: Ditto
* LayoutTests/fast/ruby/rubyDOM-remove-rt2.html: Ditto
* LayoutTests/fast/ruby/rubyDOM-remove-text1.html: Ditto
* LayoutTests/fast/ruby/rubyDOM-remove-text2.html: Ditto

Canonical link: https://commits.webkit.org/280342@main

webkit-commit-queue commented:
Committed 280342@main (21879fdc9bb7): <https://commits.webkit.org/280342@main>

Reviewed commits have been landed. Closing PR #30150 and removing active labels.
